.gitignore

```diff
@@ -127,3 +128,7 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+
+# Cppflow Stuff
+data/frameProcessor/include/cppflow
```

Comment:
If you're excluding cppflow includes here, how does the repo express a dependency on it? Should it be done with a CMake package finder (like e.g. ZeroMQ) instead?

Reply:
Likely yes, it should be done the same way we include external packages like ZeroMQ. It was placed into the include folder as a "temporary" method to get it working, but I should be able to move it out of there and find it using cmake instead